pyproject.toml

```diff
@@ -10,31 +10,32 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "3.6.*"
-authutils = "^4.0.0"
+python = "^3.6.1"
+authutils = "^6.0.0"
 cdislogging = "^1.0.0"
-cdiserrors =  "^0.1.2"
+cdiserrors =  "^1.0.0"
 doiclient = {git = "https://github.com/uc-cdis/doiclient", rev = "1.0.0"}
 dosclient = {git = "https://github.com/uc-cdis/dosclient", rev = "1.1.0"}
 gen3authz = "^1.0.4"
 hsclient = {git = "https://github.com/uc-cdis/hsclient", rev = "1.0.0"}
 indexclient = "^2.1.0"
-jsonschema = "^2.5.1"
-flask = "^1.1.1"
+jsonschema = "^3.2"
+flask = "^2.0.1"
```

Comment:
woah! ✨ 
did you check if the breaking changes from 2.0.0 would affect indexd?

Reply:
Well it passed jenkins and travis, is there anything else I should double check?
Looking at https://flask.palletsprojects.com/en/2.0.x/changes/#version-2-0-0 some of the major changes like dep support for python <=3.5 and bump up some other minimum req should be fine with what we've got.
There re some json func dep but we don't use them

Reply:
I wish I could say I trust jenkins 100% but 😆 you never know
But yeah if you read through the list and you think it's fine, it's all good